### ISSUE TITLE
SDL2: Incorrect name and value for pygame.NUMEVENTS

### ISSUE THREAD

**Issue by charlesej** [pygame/pygame#1230]
**Expected results:**
- The event name for `pygame.NUMEVENTS` should be 'Unknown'
- All event values should be greater than `pygame.NOEVENT` and less than `pygame.NUMEVENTS` (`pygame.NOEVENT` < event < `pygame.NUMEVENTS`)

**Actual results:**
- The event name for `pygame.NUMEVENTS` is 'ActiveEvent'
- Some event values are greater than `pygame.NUMEVENTS`

---

This code is used to print out some event names and values.

```python
import pygame

EVENT_DATA = (
    ('NoEvent', pygame.NOEVENT),
    ('ActiveEvent', pygame.ACTIVEEVENT),
    ('VideoResize', pygame.VIDEORESIZE),
    ('VideoExpose', pygame.VIDEOEXPOSE),
    ('Unknown', pygame.NUMEVENTS),
)

for expected_name, event in EVENT_DATA:
    name = pygame.event.event_name(event)

    if name == expected_name:
        print('{}=0x{:X}'.format(name, event))
    else:
        print('{}=0x{:X} (expected_name={})'.format(name, event, expected_name))
```

---

Output using pygame 1.9.4. The expected name for `pygame.NUMEVENTS` is 'Unknown' and the given event values are between `pygame.NOEVENT` and `pygame.NUMEVENTS`.
```
pygame 1.9.4
Hello from the pygame community. https://www.pygame.org/contribute.html
NoEvent=0x0
ActiveEvent=0x1
VideoResize=0x10
VideoExpose=0x11
Unknown=0x20
```

---

Output using pygame 2.0.0.dev3. The name for `pygame.NUMEVENTS` is 'ActiveEvent' and the given event values are not between `pygame.NOEVENT` and `pygame.NUMEVENTS`.
```
pygame 2.0.0.dev3 (SDL 2.0.10, python 3.7.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
NoEvent=0x0
ActiveEvent=0xA000
VideoResize=0xA001
VideoExpose=0xA002
ActiveEvent=0xA000 (expected_name=Unknown)
```

---

System details:
- os: windows 10 (64bit)
- python: 3.7.2 (64bit)
- pygame: 1.9.4 and 2.0.0.dev3 (SDL: 2.0.10) at 7a5ed0b864645e7c325d80aa8866e837559c7a1c

**Comment by nthykier** [comment]
This issue appears to have been fixed by #1246 and I cannot reproduce it on the master branch.

I recommend that this issue is closed as fixed/resolved by someone with permission to do so.  (e.g. @illume ?)

**Comment by charlesej** [comment]
@nthykier Thanks for looking into this.
